### PR TITLE
{chem}[foss-2020b][intel-2020b]add dl poly 4 easyconfigs

### PR DESCRIPTION
**Pull request by alinelena**

### CODE REVIEW

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
eb-u2004 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 3.8.5
See https://gist.github.com/5b7258cdbfa635a1e4985fb7a601c157 for a full test report.

**Comment by zao** [comment]
* A build dependency on `CMake` is needed.

* Is the `tar` dance with the sources necessary, can't it be handled with `filename` and/or `download_filename`?

* Normally `sources_urls` is separate from the `sources` list unless there's a need to avoid collisions.

File: easybuild/easyconfigs/d/DL_POLY_4/DL_POLY_4-5.0.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'CMakeMake'
+name =  "DL_POLY_4"
+version = "5.0.0"
+homepage = "https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx"
+description = "classical md"
```
Comment (Micket) [inline easybuild/easyconfigs/d/DL_POLY_4/DL_POLY_4-5.0.0-foss-2020b.eb:5]:
```suggestion
description = "DL_POLY is a general purpose classical molecular dynamics (MD) simulation software"
```

File: easybuild/easyconfigs/d/DL_POLY_4/DL_POLY_4-5.0.0-intel-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+easyblock = 'CMakeMake'
+name =  "DL_POLY_4"
+version = "5.0.0"
+homepage = "https://www.scd.stfc.ac.uk/Pages/DL_POLY.aspx"
+description = "classical md"
```
Comment (Micket) [inline easybuild/easyconfigs/d/DL_POLY_4/DL_POLY_4-5.0.0-intel-2020b.eb:5]:
```suggestion
description = "DL_POLY is a general purpose classical molecular dynamics (MD) simulation software"
```

**Review by Micket — CHANGES_REQUESTED** [review]
in addition to what zao wrote

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
eb-u2004 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 3.8.5
See https://gist.github.com/71a1f38194c7b56ab3f8df99560a1d0c for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node354.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/19f691531823416295fb84f59bd55347 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @alinelena !

**Pull request merged**